boilerplate/app/navigators/AppNavigator.tsx

```diff
@@ -15,12 +15,9 @@ import { StackScreenProps } from "@react-navigation/stack"
 import { observer } from "mobx-react-lite"
 import React from "react"
 import { useColorScheme } from "react-native"
+import * as Screens from "app/screens"
```

Comment:
This is a great solution!

Reply:
I get TS errors against `app/screens` though. Is that from an absolute path plugin?

Reply:
Oh yes but you can change it back to relative if that isn't set up in your project (it is in Ignite boilerplate now) yet